src/GUniLPOracle.t.sol

```diff
@@ -586,4 +592,20 @@ contract GUniLPOracleTest is DSTest {
         assertLt(sqrtPriceX96, 1 << 160);
     }
 
+    function test_sqrt_price_ratio_fuzz(uint256 p0, uint256 dec0, uint256 p1, uint256 dec1) public {
+        p0 %= MAX_PRICE;
+        if (p0 < WAD) p0 = MIN_PRICE;
```

Comment:
This should be `if (p0 < MIN_PRICE) p0 = MIN_PRICE;`

Reply:
Done in f0a714d.